index.html

```diff
@@ -0,0 +1,284 @@
+                <a href="#"
+                  ><img
+                    class="m-social-media"
+                    src="assets/medium.png"
+                    alt="medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    class="g-social-media"
+                    src="assets/github.png"
+                    alt="github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    class="v-social-media"
+                    src="assets/vec.png"
+                    alt="vector"
+                /></a>
+              </li>
+            </ul>
+            <button type="button" class="more-btn">Get my resume</button>
+          </section>
+          <section>
+            <ul class="lang">
+              <li class="lang-exp">
+                <div class="no-1">
+                  <p>Languages</p>
+                  <img src="assets/arrow_bottom.png" alt="arrow_bottom" />
+                </div>
+                <ul class="languages">
+                  <li>
+                    <img src="assets/js.png" alt="JavaScript" />
+                    <a href="#">JavaScript</a>
+                  </li>
+                  <li>
+                    <img src="assets/html.png" alt="CSS" />
+                    <a href="#">HTML</a>
+                  </li>
+                  <li>
+                    <img src="assets/css.png" alt="CSS" />
+                    <a href="#">CSS</a>
+                  </li>
+                </ul>
+              </li>
+              <li class="lang-exp">
+                <div class="no-1">
+                  <p>Frameworks</p>
+                  <img src="assets/arrow_right.png" alt="arrow_bottom" />
+                </div>
+                <hr />
+              </li>
+              <li class="lang-exp">
+                <div class="no-1">
+                  <p>Skills</p>
+                  <img src="assets/arrow_right.png" alt="arrow_bottom" />
+                </div>
+                <hr />
+              </li>
+            </ul>
+          </section>
+        </div>
+      </section>
+  
+    <section class="contact-info">
+      <article class="contact-headings"> 
+        <h2>Contact me</h2>
+         <p> If you have an application you are interested in developing, 
+           a feature that you need built or a project that needs coding.
+            I’d love to help with it  </p> 
+        </article>
+
+      <form action="https://formspree.io/f/xwkyyjgv" method="POST">
+
+      <input type="text" name="username"  maxlength="30" placeholder="Full name" required>
+
+      <input type="email" name="email" placeholder="Email" required>
+
+      <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+      <button class="form-submit" type="submit">Get It Touch </button>
+      </form>
```

Comment:
- [ ] You don't have padding from the right and there is a black border just after the get in touch buttons. Kindly implement it.
- [ ] kindly decrease the height of the text as it is covering a greater height than the Figma design.
- [ ] The get in touch buttons have bigger placeholder So fix it as well.
### your design
![his](https://user-images.githubusercontent.com/81727988/171867109-26cc8fe3-30d2-49c3-ac2c-7d068a373185.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/171867146-0d4d40fa-ca65-448c-bcab-4c4c5eb1348c.png)